### PR TITLE
fullscreen examples should have 100% height

### PR DESCRIPTION
**Pull request by jschuler**
Particularly noticeable for the page demos

Closes: https://github.com/patternfly/patternfly-react/issues/1294

### CODE REVIEW

**Comment by jschuler** [comment]
Not sure, Core doesn't mention it either, perhaps we should? @mcoker

**Comment by mcoker** [comment]
@jschuler yeah, I think we should. I don't think having it in the demo is enough for a user to figure out it's required to make the page component full height.

File: packages/patternfly-4/react-docs/src/templates/mdxTemplate.js [hunk 1]
```diff
@@ -3,15 +3,27 @@ import PropTypes from 'prop-types';
...
+import { Title, PageSection, Text, TextList, TextListItem } from '@patternfly-safe/react-core';
 import { MDXProvider } from '@mdx-js/react';
 import { MDXRenderer } from '../components/mdx-renderer';
 
 const components = {
+  wrapper: componentProps => <div className="theme-pf-c-content" {...componentProps} />,
   code: LiveEdit,
-  pre: React.Fragment
+  pre: React.Fragment,
+  p: componentProps => <Text component="p" {...componentProps} />,
```
Comment (redallen) [inline packages/patternfly-4/react-docs/src/templates/mdxTemplate.js]: We (eventually) need to make sure every HTML tag is covered here that doesn't look right inside `<div className="theme-pf-c-content">`.

File: packages/patternfly-4/react-docs/src/styles/workspace.css [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#___gatsby {
```
Comment (redallen) [inline packages/patternfly-4/react-docs/src/styles/workspace.css]: This works, but could also be solved via `wrapper: componentProps => <div className="theme-pf-c-content" style={{height: "100%"}} {...componentProps} />` in `mdxFullscreenTemplate.js` to be consistent with `mdxTemplate.js`.

Comment (jschuler) [reply packages/patternfly-4/react-docs/src/styles/workspace.css]: that would only set 100% height on the wrapper, but the ancestors #__gatsby and a child div still need 100% as well

Comment (redallen) [reply packages/patternfly-4/react-docs/src/styles/workspace.css]: Ah -- you're right!

**Pull request merged**